google/cloud/firestore_v1/async_query.py

```diff
@@ -371,12 +398,18 @@ async def _make_stream(
             if snapshot is not None:
                 yield snapshot
 
+            if metrics is None and response.explain_metrics:
+                metrics = response.explain_metrics
+                yield metrics
```

Comment:
this metrics variable seems unnecessary to me. Can we just do this?

```
if response.explain_metrics:
    yield response.explain_metrics
```

Reply:
This can prevent it from yielding `explain_metrics` more than twice. I think it's unlikely to happen, should we worry about it?

Reply:
I think that's ok. If we somehow did get multiple metrics in a stream, the consumers of this would just use the most recent value, which seems like a fine behaviour to me

Reply:
Removed all in async funcs.